src/main/java/org/store/springbootbookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package org.store.springbootbookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
for String validation use `@NotBlank`
for other places also
```suggestion
    @NotBlank
```